File: core/src/main/java/org/apache/parquet/hadoop/InternalParquetRecordReaderWrapper.java
package org.apache.parquet.hadoop;

import org.apache.parquet.filter2.compat.FilterCompat;
import org.apache.parquet.hadoop.api.ReadSupport;

public class InternalParquetRecordReaderWrapper<T> extends InternalParquetRecordReader<T> {
  public InternalParquetRecordReaderWrapper(final ReadSupport<T> readSupport,
                                            final FilterCompat.Filter filter) {
    super(readSupport, filter);
  }

  public InternalParquetRecordReaderWrapper(final ReadSupport<T> readSupport) {
    super(readSupport);
  }
}

File: core/src/main/scala/org/apache/parquet/hadoop/InternalParquetRecordReaderWrapper.scala
package org.apache.parquet.hadoop

import org.apache.parquet.hadoop.api.ReadSupport
import org.apache.parquet.filter2.compat.FilterCompat._

class InternalParquetRecordReaderWrapper[T](readSupport: ReadSupport[T], filter: Filter)
  extends InternalParquetRecordReader[T](readSupport, filter) {}
